database/seeders/BillsSeeder.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Database\Seeders;
+
+use Illuminate\Database\Console\Seeds\WithoutModelEvents;
+use Illuminate\Database\Seeder;
+
+class BillsSeeder extends Seeder
```

Comment:
About the seeders, I think we'll end not having one seeder for model, but one seeder for context
The main reason for this is due all models are too related, and to have something "real" we need to create a bunch of them